sam.c

```diff
@@ -1162,7 +1164,6 @@ enum SamError sam_cmd(Vis *vis, const char *s) {
 				break;
 			}
 		}
-		vis_mode_switch(vis, completed ? VIS_MODE_NORMAL : VIS_MODE_VISUAL);
```

Comment:
I could not reproduce a situation where it was needed, and I had to get rid of this to get things working. If needed I'll try to work around it.